packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -342,67 +371,61 @@ export const Account = ({
             ))}
           </>
         ) : (
-          <li data-testid="" className="gradient-info-display">
+          <li
+            data-testid=""
+            className={`${styleClasses.gradientInfoDisplay} ${styleClasses.li}`}
+          >
             This account is not currently signed in.
           </li>
         )}
-        <li></li>
-        <br />
-        <hr />
-        <li>
-          <h4>Account History</h4>
-          <div className="account-history-info">
-            {securityEvents && securityEvents.length > 0 ? (
-              <>
-                <TableContainer component={Paper}>
-                  <Table
-                    className="account-history-table"
-                    aria-label="simple table"
-                  >
-                    <TableHead>
-                      <TableRow>
-                        <TableCell>Event</TableCell>
-                        <TableCell>Timestamp</TableCell>
-                      </TableRow>
-                    </TableHead>
-                    <TableBody>
-                      {securityEvents.map(
-                        (securityEvents: SecurityEventsType) => (
-                          <TableRow>
-                            <TableCell>{securityEvents.name}</TableCell>
-                            <TableCell>
-                              {dateFormat(
-                                new Date(securityEvents.createdAt!),
-                                DATE_FORMAT
-                              )}
-                            </TableCell>
-                          </TableRow>
-                        )
-                      )}
-                    </TableBody>
-                  </Table>
-                </TableContainer>
-              </>
-            ) : (
-              <div
-                data-testid="acccount-security-events"
-                className="security-events"
-              >
-                No account history to display.
-              </div>
-            )}
-          </div>
-        </li>
-        <hr />
-        <DangerZone
-          {...{
-            uid,
-            disabledAt: disabledAt!,
-            email: primaryEmail, // only the primary for now
-            onCleared: onCleared,
-          }}
-        />
       </ul>
+      <hr className="border-grey-50 mb-4" />
+      <h3 className="mt-0 my-0 mb-1 text-lg">Account History</h3>
+      <div className={`${styleClasses.gradientInfoDisplay}`}>
+        {securityEvents && securityEvents.length > 0 ? (
+          <>
+            <TableContainer component={Paper}>
+              <Table className="pt-1" aria-label="simple table">
+                <TableHead>
+                  <TableRow>
+                    <TableCell>Event</TableCell>
+                    <TableCell>Timestamp</TableCell>
+                  </TableRow>
+                </TableHead>
+                <TableBody>
+                  {securityEvents.map((securityEvents: SecurityEventsType) => (
```

Comment:
I know you didn't change this but a quick win, I think this is going to complain since we're not [setting a `key`](https://reactjs.org/docs/lists-and-keys.html) on the mapped `TableRow`s.